ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: install Rust deps
+  apt:
+    name:
+      - pkg-config
+      - libssl-dev
+    state: present
+
+- name: install rustup in userspace for root
+  shell: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
```
[WARNING]: Consider using the get_url or uri module rather than running 'curl'.  If you need to use command because get_url or uri is insufficient you can add 'warn: false' to this command task or set
'command_warnings=False' in ansible.cfg to get rid of this message.

fatal: [dev-desktop.infra.rust-lang.org]: FAILED! => {"changed": true, "cmd": "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh", "delta": "0:00:00.359799", "end": "2021-12-15 18:30:07.489197", "msg": "non-zero return code", "rc": 1, "start": "2021-12-15 18:30:07.129398", "stderr": "info: downloading installer\nrustup: Unable to run interactively. Run with -y to accept defaults, --help for additional options", "stderr_lines": ["info: downloading installer", "rustup: Unable to run interactively. Run with -y to accept defaults, --help for additional options"], "stdout": "", "stdout_lines": []}
```

So probably a little more needs to be done to make this work.

Reply:
I patched the command, it works locally, but not sure about ansible.